collection-scripts/gather_monitoring

```diff
@@ -50,6 +50,32 @@ prom_get() {
       2> "$result_path.stderr"
 }
 
+prom_get_by_replica() {
+  local replica="$1"; shift
+  local object="$1"; shift
+  local path="$1"; shift
+
+  local result_path="${MONITORING_PATH}/prometheus/${path}"
+  mkdir -p "$(dirname "${result_path}")"
+
+  oc --insecure-skip-tls-verify exec pod/${replica} \
```

Comment:
Is there no way to prevent using the `--insecure-skip-tls-verify` option?

Reply:
@davemulford yep. Removed!